.github/workflows/ci.yml

```diff
@@ -38,6 +48,11 @@ jobs:
       - name: Install dependencies
         run: bun install --frozen-lockfile
 
+      - name: Install Python dependencies
+        run: |
+          cd packages/evaluatorq-py
+          uv sync --frozen --all-groups
+
```

Comment:
Can this be in the project.json of the python project? A dependency of the build command? You already have an install command there 😊 

https://nx.dev/docs/guides/tasks--caching/defining-task-pipeline#define-dependencies-between-tasks